components/confirmations.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div class="confirmations">
     <span class="confirmations-num">
-      {{ maxHeight - height }}
+      {{ (maxHeight - height) > 0 ? maxHeight - height : 0 }}
```

Comment:
does this fix #87 ?

Reply:
yeah, I think so. cool! =)

Reply:
it will force the confirmation number to be min of 0